metatron-app/metatron-discovery
# F#2437 additional metadata source info

### Description

Add  addtional metadata information after explore
**Related Issue** : 

#2437 

### How Has This Been Tested?

1. Go to Explore Data Page using lnb
2. Click Data Card
3. Check Modal shows proper data
4. Click See all list Button
5. Check Modal shows proper data

#### Need additional checks?

### Types of changes



### Checklist:


### Additional Context
